src/cascadia/TerminalApp/SnippetsPaneContent.xaml

```diff
@@ -0,0 +1,262 @@
+                                            </VisualState.Setters>
+                                        </VisualState>
+                                    </VisualStateGroup>
+
+                                </VisualStateManager.VisualStateGroups>
+                            </Border>
+                        </ControlTemplate>
+                    </Setter.Value>
+                </Setter>
+
+            </Style>
+
+            <DataTemplate x:Key="TaskItemTemplate"
+                          x:DataType="local:FilteredTask">
+                <mux:TreeViewItem x:Name="rootItem"
+                                  AutomationProperties.Name="{x:Bind Item.Name, Mode=OneWay}"
+                                  ItemsSource="{x:Bind Children}"
+                                  KeyUp="_treeItemKeyUpHandler"
+                                  Visibility="{x:Bind Visibility, Mode=OneWay}">
+                    <Grid AutomationProperties.AccessibilityView="Raw">
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="*" />
+                            <!--  Name  -->
+                            <RowDefinition Height="*" />
+                            <!--  Input  -->
+                            <RowDefinition Height="0" />
+                            <!--  Input gets put here when we're a nested command, to hide it  -->
+                        </Grid.RowDefinitions>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="Auto" />
+                            <ColumnDefinition Width="*" />
+                        </Grid.ColumnDefinitions>
+
+
+                        <Button x:Uid="SnippetPlayButton"
+                                Grid.Row="0"
+                                Grid.RowSpan="1"
+                                Grid.Column="0"
+                                Margin="-28,0,0,0"
+                                Padding="3"
+                                VerticalAlignment="Bottom"
+                                Background="Transparent"
+                                BorderBrush="Transparent"
+                                Click="_runCommandButtonClicked"
+                                Style="{StaticResource PlayButtonTemplate}"
+                                Visibility="{x:Bind mtu:Converters.InvertedBooleanToVisibility(HasChildren), Mode=OneWay}">
+
+                            <Button.Content>
+                                <FontIcon FontFamily="{ThemeResource SymbolThemeFontFamily}"
+                                          FontSize="12"
+                                          Glyph="&#xE768;" />
+                                <!--
+                                    xE768 is Play, which is just an outline.
+                                    xF5B0 is PlaySolid, also a good option. Seemed
+                                    better to have a lightweight outline though
+                                -->

```

Comment:
Lightweight looks nice. Please don't change haha.

FYI, there is a chance that the play button gets flagged in an a11y review. It doesn't have an outline for the button and I could see them getting upset about that. I could also see us getting flagged for contrast issues. IMHO, let's keep the current design. I like it. If they flag it, we'll handle it then.

Reply:
thank you! lemme tell you this:

![b9cd048d-a1bd-4c4f-ab45-30db7d89d664](https://github.com/microsoft/terminal/assets/18356694/837390aa-7b4c-41ff-927d-130c6471f5da)

took entirely too long in XAML